src/main/java/com/cloudbees/jenkins/support/SupportPlugin.java

```diff
@@ -763,6 +774,33 @@ public boolean accept(File dir, String name) {
             }
         }
 
+        private void cleanupOldBundles(File bundleDir, File justGenerated) {
+            thread.setName(String.format("%s periodic bundle generator: tidying old bundles since %s",
+                    SupportPlugin.class.getSimpleName(), new Date()));
+            File[] files = bundleDir.listFiles(new FilenameFilter() {
+                public boolean accept(File dir, String name) {
+                    return name.startsWith(SupportPlugin.getBundlePrefix()) && name.endsWith(".zip");
```

Comment:
> if someone has taken the habit to put unrelated files there

Well don’t do that then.

I agree it would be better to just delete all `*.zip`.